components/compliance-service/reporting/relaxting/reports.go

```diff
@@ -1123,7 +1139,20 @@ func (backend ES2Backend) getFiltersQuery(filters map[string][]string, latestOnl
 	if len(filters["start_time"]) > 0 || len(filters["end_time"]) > 0 {
 		endTime := firstOrEmpty(filters["end_time"])
 		startTime := firstOrEmpty(filters["start_time"])
+		layout := "2006-01-02T15:04:05Z"
+		endtime, err := time.Parse(layout, endTime)
+		starttime, err := time.Parse(layout, startTime)
+		if err != nil {
+			panic(err)
+		}
+		diff := endtime.Sub(starttime).Hours() / 24
 
+		if diff > 90 {
+			logrus.Error("Range should be less than 90 days")
```

Comment:
The error should not only be logged but also responded back to the API. We cannot continue if the diff is more than 90 days